docs/desktop-wpf/migration/convert-project-from-net-framework.md

```diff
@@ -54,43 +54,45 @@ Older .NET Framework projects typically list their NuGet dependencies in a *pack
 
 When migrating, there are two advantages to using `<PackageReference>`-style references:
 
-1. This is the style of NuGet reference that is required for the new .NET Core project file. If you're already using `<PackageReference>`, those project file elements can be copied and pasted directly into the new project.
-1. Unlike a *packages.config* file, `<PackageReference>` elements only refer to the top-level dependencies that your project depends on directly. All other transitive NuGet packages will be determined at restore time and recorded in the autogenerated *obj\project.assets.json file*. This makes it much easier to reason about what dependencies your project has, which is useful when determining whether the necessary dependencies will work on .NET Core or not.
+- This is the style of NuGet reference that is required for the new .NET Core project file. If you're already using `<PackageReference>`, those project file elements can be copied and pasted directly into the new project.
+- Unlike a packages.config file, `<PackageReference>` elements only refer to the top-level dependencies that your project depends on directly. All other transitive NuGet packages will be determined at restore time and recorded in the autogenerated obj\project.assets.json file. This makes it much easier to determine what dependencies your project has, which is useful when determining whether the necessary dependencies will work on .NET Core or not.
 
-The first step to migrating a .NET Framework app to .NET Core is to update it to use `<PackageReference>` NuGet references. Visual Studio makes this simple. Just right-click on the project's *packages.config* file in Visual Studio's **Solution Explorer** and select **Migrate packages.config to PackageReference**.
+The first step to migrating a .NET Framework app to .NET Core is to update it to use `<PackageReference>` NuGet references. Visual Studio makes this simple. Just select the project's *packages.config* file in Visual Studio's **Solution Explorer**, and then right-click **Migrate packages.config to PackageReference**.
```

Comment:
I think `right-click` and `select` are now reversed. The old sentence seems correct to me.